app/chapters/[slug]/transacting/transacting-2/page.tsx

```diff
@@ -57,6 +58,14 @@ export default function Genesispt2() {
             className="genesis-p2"
             dangerouslySetInnerHTML={{ __html: genesis.body.html }}
           ></div>
+          <div className='genesis-p2'>
+            <pre>
+              <code className='language-shell flex justify-between items-center'>
```

Comment:
Same padding classes as in `genesis-pt2`?

Reply:
Just added 1f1ed38054880108b0321763d8828a71de13868d and 8c8298aba4368a9b4f42f656ea9e730e252e3297 for this.
They wrap the copyable code into a reusable component.